src/Node/Express/Request.js

```diff
@@ -138,3 +138,15 @@ exports._getOriginalUrl = function (req) {
     };
 };
 
+exports._getData = function (req, field) {
+    return function () {
+        return req[field];
+    };
+};
+
+exports._setData = function (req, field, val) {
+    return function () {
+        req[field] = val;
```

Comment:
I had this idea and thought that if I do the way I did, it would be a good way to "hack into" the request object if necessary. Giving it a second thought now I don't think I was right. I'll amend the PR.